main.py

```diff
@@ -163,12 +165,78 @@ def get_spool_client():
 
 
 # pylint: disable=unused-argument
```

Comment:
Could you keep this line directly above the first Cloud Function, the same as it was before your change? It's related to those only.

It would help to also add a comment explaining why it's there (because we don't use/need Cloud Function arguments).

Reply:
okay